src/waveform/renderers/waveformmark.cpp

```diff
@@ -59,4 +60,5 @@ void WaveformMark::setKeyAndIndex(const ConfigKey& key, int i) {
     DEBUG_ASSERT(m_pPointCos == NULL);
     m_pPointCos = new ControlProxy(key);
```

Comment:
By including "util/memory.h" from Mixxx and using std::unique_ptr you can replace this operator new with the following C++14 construct:
m_pPointCos = std::make_unique<ControlProxy>(key);

std::make_unique is missing in C++11 so we have added our own version